NextcloudTalk/Calls/CallViewController.swift

```diff
@@ -1106,13 +1108,26 @@ class CallViewController: UIViewController,
 
         guard self.room.type != .oneToOne, let personImage = UIImage(systemName: "person.2") else { return }
 
-        DispatchQueue.main.async {
-            let participantAttachment = NSTextAttachment(image: personImage.withTintColor(self.participantsLabel.textColor))
-            let participantText = NSMutableAttributedString(attachment: participantAttachment)
-            participantText.append("  \(self.peersInCall.count + 1)".withFont(self.participantsLabel.font))
+        WebRTCCommon.shared.dispatch {
+            var participantsInCall: [TalkActor] = []
+
+            self.peersInCall.forEach { peerConnection in
+                let actor = self.callController?.getActorFromSessionId(peerConnection.peerId) ?? TalkActor()
+                participantsInCall.append(actor)
+            }
```

Comment:
```suggestion
            var participantsInCall = self.peersInCall.compactMap { self.callController?.getActorFromSessionId($0.peerId ) }
```

Should be the same I think.